modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -75,11 +75,52 @@ function inope_policy_review_permission() {
 }
 
 /**
+ * Implements hook_form_alter().
+ */
+function inope_policy_review_form_alter(&$form, &$form_state, $form_id) {
+
+  global $user;
+  $account = $user;
+
+  // Store argument from the autoloader in hook_menu().
+  $node = $form_state['build_info']['args'][0];
+
+  // Check that we are only altering the inope policy review form.
+  if ($form_id == 'inope_policy_review_form') {
+
+    // Query the policy review table for any records related to the current
+    // node, unapproved notes, and are from the current user.
+    $results = db_select('policy_review', 'pr')
+      ->fields('pr')
+      ->condition('nid', $node->nid, '=')
+      ->condition('note_status', 0, '=')
+      ->condition('uid', $account->uid, '=')
+      ->execute()
+      ->fetchAssoc();
+
+    // If the active user has submitted something and is not an administrator
+    // clear the form items we don't want to show and show a message.
+    if (!empty($results) && !in_array('administrator', $account->roles)) {
```

Comment:
You have a couple options:
1) You could see if the form needs to be rendered at all in the page callback
2) You could conditionally not add these items in the first place based on if this needs to be rendered in the first place -- that's probably prone to caching problems though.  That may be why you were using alters. hmm, - I guess I'd probably come down on the side of 1 then -- because if work doesn't need to be done because of a business rule -- a good rule of thumb is to not do the work.